cea/technologies/thermal_network/simplified_thermal_network.py

```diff
@@ -236,7 +238,7 @@ def thermal_network_simplified(locator, config, network_name):
 
         # add loads
         building_base_demand_m3s = {}
-        for building in volume_flow_m3pers_building.keys():
+        for building in building_names:
```

Comment:
it is the same - `volume_flow_m3pers_building` is created a few lines above with one entry per `building in building_names`. I changed it because who can ever remember what the keys were? `building_names` is much clearer.